zbus/src/connection/socket/mod.rs

```diff
@@ -8,6 +8,8 @@ pub use split::{BoxedSplit, Split};
 
 #[cfg(unix)]
 mod command;
+#[cfg(unix)]
+pub(crate) use command::Command;
```

Comment:
This ended up in the wrong commit. Please move it to the previous one that adds the type.

Reply:
Good catch. Should be fixed in latest revision.

Reply:
Thanks.